articles/config-less-fleetd-agent-deployment.md

```diff
@@ -0,0 +1,88 @@
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>PayloadContent</key>
+    <array>
+      <dict>
+        <key>EnrollSecret</key>
+        <string>YOUR_ENROLL_SECRET_HERE</string>
+        <key>FleetURL</key>
+        <string>YOUR_FLEET_URL_HERE</string>
+        <key>PayloadDisplayName</key>
+        <string>Fleetd configuration</string>
+        <key>PayloadIdentifier</key>
+        <string>com.fleetdm.fleetd.config</string>
+        <key>PayloadType</key>
+        <string>com.fleetdm.fleetd.config</string>
+        <key>PayloadUUID</key>
+        <string>476F5334-D501-4768-9A31-1A18A4E1E807</string>
+        <key>PayloadVersion</key>
+        <integer>1</integer>
+      </dict>
+    </array>
+    <key>PayloadDisplayName</key>
+    <string>Fleetd configuration</string>
+    <key>PayloadIdentifier</key>
+    <string>com.fleetdm.fleetd.config</string>
+    <key>PayloadType</key>
+    <string>Configuration</string>
+    <key>PayloadUUID</key>
+    <string>0C6AFB45-01B6-4E19-944A-123CD16381C7</string>
+    <key>PayloadVersion</key>
+    <integer>1</integer>
+    <key>PayloadDescription</key>
+    <string>Default configuration for the fleetd agent.</string>
+  </dict>
+</plist>
+```
+
+
+## For Windows:
+
+1. Download the Base MSI installer from [https://download.fleetdm.com/fleetd-base.msi](https://download.fleetdm.com/fleetd-base.msi) (once installed, `fleetd` and `fleet-desktop` will be upgraded to the latest)
+
+2. Install fleet on Windows boxes by passing the `FLEET_URL` and `FLEET_SECRET` properties to the MSI installer:
+
+```xml
+msiexec /i fleetd-base.msi FLEET_URL="<target_url>" FLEET_SECRET="<secret_to_use>"
+```
+
+These steps are a flexible alternative to deploying Fleet's agent across macOS and Windows platforms. This method, focused on separating the configuration from the `fleetd` package, empowers you with more control and simplifies the management of your device enrollments.
+
+This approach complements the original packaging method, allowing you to choose the best fit for your organization’s needs. Whether you prioritize streamlined package generation or prefer granular control over configuration distribution, these methods foster an open, flexible environment for deploying Fleet.
+
+We encourage you to explore this alternative method in your environment and see how it aligns with your operational workflows. If you have any questions, insights, or experiences to share, feel free to join our community [Fleet Slack channels](https://fleetdm.com/support). Your feedback helps us improve and fosters a collaborative space where ideas and solutions can flourish.
+
+
+<meta name="articleTitle" value="Windows MDM setup">
+<meta name="authorFullName" value="Noah Talerman">
+<meta name="authorGitHubUsername" value="noahtalerman">
```

Comment:
```suggestion
<meta name="articleTitle" value="Config-less fleetd agent deployment">
<meta name="authorFullName" value="JD Strong">
<meta name="authorGitHubUsername" value="spokanemac">
```

Reply:
This should remain in Noah's name. I'll update the title.